tools/daemon/message.c

```diff
@@ -173,6 +173,12 @@ int proc_message(void)
 			rc = proc_msg_exit(msg_hdr, len);
 			break;
 		case VMA_MSG_FLOW:
+			/* Note: special loopback logic, it
+			 * should be added first as far as observed issue with delay
+			 * in activation loopback filters in case two processes
+			 * communicate locally w/o SRIOV
+			 */
+			proc_msg_flow(msg_hdr, len, NULL);
```

Comment:
Why not checking if (data->type == VMA_MSG_FLOW_EGRESS) and only then calling to the function ?
Is it very hard to track it...

Reply:
Technically it can be done but I think all check should be done inside special function that proceeds flow logic.